site/static/examples/tas/sample-queues.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
   name: "tas-flavor"
 spec:
   nodeLabels:
-    cloud.provider.com/node-group: "tas-group"
+    cloud.provider.com/node-group: "tas-node-group"
```

Comment:
nit, please revert and submit a dedicated PR if this is beneficial, but I remember a recent PR going in the opposite way.